apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -108,6 +115,22 @@ export default class PublishAction extends BaseRushAction {
       `the package is older than the current latest, so in publishing workflows for older releases, providing ` +
       `a tag is important. When hotfix changes are made, this parameter defaults to 'hotfix'.`
     });
+
+    // NPM pack tarball related parameters
+    this._pack = this.defineFlagParameter({
+      parameterLongName: '--pack',
+      description:
+        `Packs projects into tarballs instead of publishing to npm repository. It can only be used when ` +
+        `--include-all is specified. If this flag is specified, NPM registry related parameters will be ignored.`
```

Comment:
>parameters will be ignored. [](start = 85, length = 27)

Should we report an error for invalid parameter combinations?

Yesterday you mentioned that a number of these options are deprecated and will be removed soon.  I wouldn't waste much time validating those.  But for options that we expect to be part of the long-term design for "rush publish", we should include validation to avoid mistakes/confusion.  It can be very frustrating to specify a flag and not understand why it isn't doing anything.  (This was a main motivation for why we validate parameters at all heheh.)